save-frontend/webpack.config.d/optimizations.js

```diff
@@ -1,11 +1,3 @@
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-
-config.plugins.push(
-    new HtmlWebpackPlugin({
-        template: 'index.html'
```

Comment:
Problems with cache are back in 3.. 2..

Why did you need to remove it? Please describe the problem with the dropdown more specifically. `HtmlWebpackPlugin` is needed to resolve an EJS expression in `index.html` (inject current timestamp into a `<meta>` tag) to make file content refresh on every build and avoid stale cache. Which problems does it code with the existing code?

Reply:
The dropdown has the _ariaExpanded_ parameter, which changes to _true_ on click and shows a dropdown list. Now _ariaExpanded_ doesn't change on click.

Reply:
Any clues about why this is happening? `HtmlWebpackPlugin` is called only during build time and in theory only affects `index.html`, where there are no references to Bootstrap or to Popper.

Reply:
> Any clues about why this is happening? `HtmlWebpackPlugin` is called only during build time and in theory only affects `index.html`, where there are no references to Bootstrap or to Popper.

We have no idea :(

Reply:
Changed the logic for displaying Dropdown. It works now. But I still don't understand why webpack broke it.